### PR TITLE
chore(deps): update to backstage 1.26.x

### PR DESCRIPTION
**Pull request by schultzp2020**
## Description

Please explain the changes you made here.

## Which issue(s) does this PR fix

- Fixes #?

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

**Review by nickboldt — COMMENTED** [review]
skimmed, but 113 files (including a lot of yarn locks) is too much to manually review.

looks good, especially since I see you're switching to the new backstage-community versions of deps.

File: dynamic-plugins/imports/package.json [hunk 1]
```diff
@@ -12,26 +12,26 @@
...
+    "@janus-idp/backstage-plugin-aap-backend": "1.6.6",
+    "@janus-idp/backstage-plugin-acr": "1.3.6",
+    "@janus-idp/backstage-plugin-analytics-provider-segment": "1.3.6",
+    "@janus-idp/backstage-plugin-jfrog-artifactory": "1.3.6",
+    "@janus-idp/backstage-plugin-keycloak-backend": "1.9.6",
+    "@janus-idp/backstage-plugin-nexus-repository-manager": "1.5.6",
+    "@janus-idp/backstage-plugin-ocm": "4.0.2",
+    "@janus-idp/backstage-plugin-ocm-backend": "4.0.2",
+    "@janus-idp/backstage-plugin-quay": "1.6.7",
+    "@janus-idp/backstage-plugin-rbac": "1.18.2",
```
Comment (nickboldt) [inline dynamic-plugins/imports/package.json:29]: should we have rbac-node, rbac-common, and rbac-backend here too?

File: packages/app/package.json [hunk 2]
```diff
@@ -14,50 +14,46 @@
...
+    "@backstage/plugin-scaffolder": "1.19.3",
+    "@backstage/plugin-search": "1.4.10",
+    "@backstage/plugin-search-react": "1.7.10",
+    "@backstage/plugin-user-settings": "0.8.5",
+    "@backstage/theme": "0.5.3",
     "@emotion/react": "11.11.3",
     "@emotion/styled": "11.11.0",
     "@internal/plugin-dynamic-plugins-info": "*",
-    "@janus-idp/backstage-plugin-rbac-common": "1.4.0",
+    "@janus-idp/backstage-plugin-rbac-common": "1.4.2",
```
Comment (nickboldt) [inline packages/app/package.json:40]: do we need rbac-node and rbac-backend here too?

File: dynamic-plugins/imports/package.json (hunk 1 above)
Comment (Zaperex) [reply dynamic-plugins/imports/package.json:29]: Those are already statically installed in the showcase.
https://github.com/janus-idp/backstage-showcase/blob/3c9376cc455d6e014b1337f7ca1f9407f92ef371/packages/backend/package.json#L41-L42

File: packages/app/package.json (hunk 2 above)
Comment (Zaperex) [reply packages/app/package.json:40]: This is the package for the frontend, so we won't need the backend dependencies here.

Comment (schultzp2020) [reply packages/app/package.json:40]: yes

File: dynamic-plugins/imports/package.json (hunk 1 above)
Comment (schultzp2020) [reply dynamic-plugins/imports/package.json:29]: no, it doesn't need to be dynamic.

**Pull request merged**